src_assets/common/assets/web/config.html

```diff
@@ -765,11 +765,18 @@ <h1 class="my-4">Configuration</h1>
       </div>
     </div>
   </div>
-  <div class="alert alert-success my-4" v-if="success">
-    <b>Success!</b> Restart Sunshine to apply changes
+  <div class="alert alert-success my-4" v-if="saved && restart_supported">
```

Comment:
nitpick: Probably a bit reduntant to make 3 duplicated divs for the alert. You can use the <span> tag to wrap the text and then use v-if on the span themselves.